### ISSUE TITLE
Morrowind - MGE XE Mod Not Integrating Into Morrowind

### ISSUE THREAD

**Issue by trymeouteh** [ValveSoftware/Proton#3193]
# Compatibility Report
- The Elder Scrolls III: Morrowind
- 22320

## System Information
- Proton version:4.2-9

The popular MGE XE mod does not work in Proton nor does it work in Wine. The only thing from MGE XE that seems to work is the high resolution as I am able to set the games resolution to 1920x1080 when the game without MGE XE only supports up to 1024x768.

The distant land wizard crashes during distand land generation, but not only that many other MGE components do not integrate into the game such as the 3rd person camera position angle, MWSE support.

This following log file is a test showing you MGE XE does not work and when I try to load a game using a mod that uses MWSE, an error occurs saying the mod does not work due to MWSE not being present in the game.

Here is the proton log file
https://privatebin.net/?d699b621571b2156#DNBBoA5Kdd1PqtoxiVBvAUghzmVqARajVc4u2A6np2Gb

You can download MGE XE from here
https://www.nexusmods.com/morrowind/mods/41102/

Wine Bugs Reported
https://bugs.winehq.org/show_bug.cgi?id=47940
https://bugs.winehq.org/show_bug.cgi?id=39199

**Comment by trymeouteh** [comment]
https://bugs.winehq.org/show_bug.cgi?id=47940

The Wine bug report has gotten MGE XE to mostly work in Wine for the retail version of Morrowind, I did do the exact same treaks to the Morrowind Steam version and I got MGE XE to work! Except the game seems to crash after 2-5 minutes unlike the retail version in Wine.

Here is the Proton log.

https://privatebin.net/?f5f5153330e586e5#3FWWwBBBznHRnDXAJ1SUJeDitXXQFoDcQLM46MpDy2HM
